src/main/java/org/broadinstitute/hellbender/tools/copynumber/utils/CombineSegmentBreakpoints.java

```diff
@@ -0,0 +1,165 @@
+
+        final List<SimpleAnnotatedGenomicRegion> segments1 = AnnotatedRegionParser.readAnnotatedRegions(segmentFiles.get(0), columnsOfInterest);
+        final List<SimpleAnnotatedGenomicRegion> segments2 = AnnotatedRegionParser.readAnnotatedRegions(segmentFiles.get(1), columnsOfInterest);
+
+        // Check to see if we should warn the user that one or more columns of interest were not seen in any input file.
+        final Set<String> allSeenAnnotations = Sets.union(segments1.get(0).getAnnotations().keySet(), segments2.get(0).getAnnotations().keySet());
+        final Set<String> unusedColumnsOfInterest = Sets.difference(columnsOfInterest, allSeenAnnotations);
+        if (unusedColumnsOfInterest.size() > 0) {
+            final List<String> missingColumns = new ArrayList<>(unusedColumnsOfInterest);
+            throw new UserException.BadInput("Some columns of interest specified by the user were not seen in any input files: " + StringUtils.join(missingColumns, ", "));
+        }
+
+        // Create a map of input to output headers.  I.e. the annotation in the segment1 to the output that should be written in the final file.
+        //  This assumes that the keys in each entry of the list is the same.
+        // TODO: If we want to support more than two segment files, this is the only bit that requires thinking.  Once this is solved, then this logic can go into a utility class.
+        final Set<String> intersectingAnnotations = Sets.intersection(segments1.get(0).getAnnotations().keySet(), segments2.get(0).getAnnotations().keySet());
+
+        // Create the obvious mappings that are identity then tack on new annotations for conflicts.
+        // These are mappings that take the header name from the segment files and map to an output header to avoid conflicts.
+        final Map<String, String> input1ToOutputHeaderMap = segments1.get(0).getAnnotations().keySet().stream().filter(a -> !intersectingAnnotations.contains(a))
+                .collect(Collectors.toMap(Function.identity(), Function.identity()));
+        Utils.stream(intersectingAnnotations.iterator()).forEach(a -> input1ToOutputHeaderMap.put(a, a + "_" + segmentFileLabels.get(0)));
+
+        final Map<String, String> input2ToOutputHeaderMap = segments2.get(0).getAnnotations().keySet().stream().filter(a -> !intersectingAnnotations.contains(a))
+                .collect(Collectors.toMap(Function.identity(), Function.identity()));
+        Utils.stream(intersectingAnnotations.iterator()).forEach(a -> input2ToOutputHeaderMap.put(a, a + "_" + segmentFileLabels.get(1)));
+
+        final List<SimpleAnnotatedGenomicRegion> finalList = annotateCombinedIntervals(segments1, segments2,
+                Arrays.asList(input1ToOutputHeaderMap, input2ToOutputHeaderMap), getBestAvailableSequenceDictionary());
+
+        // TODO: Capture sample names in the comments if possible.
+        AnnotatedRegionParser.writeAnnotatedRegionsAsTsv(finalList, outputFile, Collections.emptyList());
```

Comment:
I'd prefer that you just add this capability, or remove the passing of comment strings for now.